lazypredict/Supervised.py

```diff
@@ -280,7 +282,21 @@ def fit(self, X_train, X_test, y_train, y_test):
             ]
         )
 
-        for name, model in tqdm(CLASSIFIERS):
+        # Here is for issue #114, if you do not want to choose all classifiers
+        # you can call LazyClassifier as shown below: 
+        # LazyClassifier(classifiers=["DecisionTreeClassifier", "RandomForestClassifier"])
+        temp_list = []
+        if self.classifiers is not CLASSIFIERS:
+            for name, model in all_estimators(): # an example of all_estimators() ↓↓↓↓
```

Comment:
I don't like this, it's running `for classifier in self.classifiers:` for every model inside `all_estimators()` resulting in a complexity of O(n^2).

Perhaps, instead of passing a list of strings, like `classifiers=["DecisionTreeClassifier", ...]`, the user should be able to pass a list of estimators with desired hyper parameters, like `classifiers=[DecisionTreeClassifier(max_depth=3), ...]`.

That way, we only need to iterate through the list of estimators and transform into a list of tuples, with something like this:
```
temp_list = []
for classifier in self.classifiers:
    full_name = (classifier.__class__.__name__, classifier)
    temp_list.append(full_name)
self.classifiers = temp_list
```

What do you think @shankarpandala?